src/python/WMCore/Storage/SiteLocalConfig.py

```diff
@@ -39,11 +39,11 @@ def loadSiteLocalConfig():
             msg = "%s env. var. provided but not pointing to an existing file, ignoring." % overVarName
             logging.log(logging.ERROR, msg)
 
-    defaultPath = "$CMS_PATH/SITECONF/local/JobConfig/site-local-config.xml"
+    defaultPath = "$SITECONFIG_PATH/JobConfig/site-local-config.xml"
```

Comment:
IIRC the `../local..` part from this path here is usually a sym. link related to the site name/setup. Are we sure it has been properly expanded in the `$SITECONFIG_PATH` env. variable? Or should we even care to check, actually?... Maybe we just take it for granted and use what ever provided inside the environment? (just a thought here, no request for a change).

Reply:
@todor-ivanov we can take it from granted, it is handled in the cms-common script below:

https://github.com/cms-sw/cms-common/blob/9c8203bf637b153697b855dfa522d6741379bcc9/cmsset_default.sh#L40-L44

Reply:
thanks @khurtado, indeed we can.